modules/distr-tracing-tempo-install-tempomonolithic-cli.adoc

```diff
@@ -6,10 +6,10 @@
 [id="distr-tracing-tempo-install-tempomonolithic-cli_{context}"]
 = Installing a TempoMonolithic instance by using the CLI
 
-:FeatureName: The TempoMonolithic instance
```

Comment:
Callout 8 description in `Installing a TempoMonolithic instance by using the web console` and `Installing a TempoMonolithic instance by using the CLI` have  TempoMonolithic instances that needs backticks.